### DISCUSSION TITLE
How to set locale?

### DISCUSSION THREAD

**Discussion by LinArcX** [leo-arch/clifm#278] [Q&A] [answered]
Every time that i run the clifm, it shows me this warning:
> clifm: Locale is not UTF-8. To avoid encoding issues you might want to set an UTF-8 locale. Ex: export LANG=es_AR.UTF-8

I set this env in my `.bashrc`. i don't know why it still complaining. any idea?

OS: windows 10, msys2
Shell: bash
Terminal Emulator: wezterm

**Comment by LinArcX** [comment, marked as answer]
Ok, I should add it to the environment variables in windows not in `.bashrc`.

But wait. i compiled clifm for windows with msys2. i should not look for envs in .bashrc first and then go to windows envs?

**Reply by leo-arch** [reply]
A program inherits the environment from its parent process (whichever it is): there's no need to check `.bashrc` nor any other file, but just query the environment for the corresponding variable via **getenv**(3), as follows: `getenv("VAR")`, in this case, `getenv("LANG")`. And this is what we do. If the call returns NULL, then the variable isn't set.

To make sure, just run `env` before running **clifm**.
